src/delegate.erl

```diff
@@ -167,6 +193,9 @@ invoke_no_result(Pid, FunOrMFA) when is_pid(Pid) ->
     ok;
 invoke_no_result([], _FunOrMFA) -> %% optimisation
     ok;
+invoke_no_result([Pid], FunOrMFA = {gen_server2, _F, _A}) when is_pid(Pid) -> %% optimisation
```

Comment:
Why is `gen_server2` hardcoded here?

Reply:
It does not affect the operation that must go through the 'delegate', for example：
{Mem, _BadNodes} = delegate:invoke(Pids, {erlang, process_info, [memory]})

In this case, you have to go through an  'delegate'.  "gen_server2 hardcoded here" is to limit modification impact to sending messages

Reply:
In other words, we only want to optimize for `gen_server2` processes?